src/hooks/transactions/useGetPollingInterval.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  CROSS_SHARD_ROUNDS,
+  TRANSACTIONS_STATUS_POLLING_INTERVAL_MS
+} from 'constants/transactionStatus';
+import { useSelector } from 'reduxStore/DappProviderContext';
+import { roundDurationSelectorSelector } from 'reduxStore/selectors';
+
+export const useGetPollingInterval = () => {
+  const roundDuration = useSelector(roundDurationSelectorSelector);
+
+  if (!roundDuration) {
```

Comment:
roundDuration == null

Reply:
Better with !, because we must skip 0 as well. roundDuration cannot be 0